pkg/storage/claim_storage_provider.go

```diff
@@ -1,9 +1,11 @@
-package claims
+package storage
 
-import "context"
+import (
+	"context"
+)
 
-// Provider is an interface for interacting with Porter's claim data.
-type Provider interface {
+// ClaimProvider is an interface for interacting with Porter's claim data.
+type ClaimProvider interface {
```

Comment:
Do you mind to explain why we use `claim` here?

Reply:
That makes perfect sense. I just wanted to make sure I understand the scope of this PR. Thanks for clarifying it!